server/engine/telemetry.go

```diff
@@ -34,6 +35,21 @@ func BuildSegmentPropertiesMap(db *gorm.DB) analytics.Properties {
 	return propertiesMap
 }
 
+func GetMetricFromMainOutputs(db *gorm.DB) {
+
+	//var outputsMap map[string]interface{}
+	var outputs []model.Output
+	db.Find(&outputs)
+	for _, data := range outputs {
+		if data.ModuleName == "" {
+			location := data.Values["location"].(map[string]interface{})["value"]
```

Comment:
This block needs a couple of checks to ensure we don't panic when data does not contain `location` and/or `access`.

Reply:
Added if checks for region and accessType